pkg/resourceinterpreter/customizedinterpreter/customized.go

```diff
@@ -45,8 +49,23 @@ func NewCustomizedInterpreter(informer genericmanager.SingleClusterInformerManag
 	if err != nil {
 		return nil, err
 	}
+
+	serviceInformer := informers.NewSharedInformerFactory(kubernetes.NewForConfigOrDie(config), 0).Core().V1().Services()
```

Comment:
Why not use the input parameters `informer` directly?

Reply:
Because it is a dynamic informer, but we need a versioned informer.  
Maybe I can use indexer to build a service lister directly, but that need to add a new method to get indexer in interface SingleClusterInformerManager, I don't know if it would be better.

Reply:
ok, i get it.

Reply:
I think we can use the versioned informer to replace the dynamic informer, if this works, we can do it in the next pr.